components/post_view/message_attachments/message_attachment/message_attachment.tsx

```diff
@@ -77,6 +77,7 @@ export default class MessageAttachment extends React.PureComponent<Props, State>
 
         this.imageProps = {
             onImageLoaded: this.handleHeightReceived,
+            onImageHeightChanged: this.checkPostOverflow,
```

Comment:
Are you sure this is needed here? I'm pretty sure we should know the dimensions of the image because the app should have received those in the post metadata.

Reply:
Oh, I see. This is called when a collapsed image is toggled, is that correct? I guess we never thought to add that when we made Markdown images collapsible.